src/GitHub/Endpoints/Repos.hs

```diff
@@ -172,6 +173,12 @@ createRepoR :: NewRepo -> Request 'RW Repo
 createRepoR nrepo =
     command Post ["user", "repos"] (encode nrepo)
 
+-- | Fork an existing repository.
+-- See <https://developer.github.com/v3/repos/forks/#create-a-fork>
+forkExistingRepoR :: Name Owner -> Name Repo -> Maybe (Name Owner) -> Request 'RW Repo
+forkExistingRepoR owner repo morg =
+    command Post ["repos", toPathPart owner, toPathPart repo, "forks" ] (maybe mempty encode morg)
```

Comment:
ah, the third argument seems to be *query string parameter*, if I read the documentation right.
Yet, we don't have support for that (`Command` constructor doesn't take `QueryString`).

It's up to you to decide, whether you
- add a TODO: note that third argument is unused
- alter `SimpleRequest`, I don't think there are anything complicated, compiler should guide you.